[docs] Expand the EAS Metadata documentation (#19028)
expo/expo: docs/pages/eas/metadata/index.md

---
title: EAS Metadata
sidebar_title: Introduction
hideTOC: true
---

import { BoxLink } from '~/ui/components/BoxLink';
import { Callout } from '~/ui/components/Callout';
import { Terminal } from '~/ui/components/Snippet';

<Callout type="warning">
  EAS Metadata is in beta and subject to breaking changes.
</Callout>
<br />

**EAS Metadata** enables you to automate and maintain your app store presence from the command line.

You need to provide a lot of information to multiple app stores before your users can use your app. This information is often about complex topics that don't apply to your app. You have to start a lengthy review process after providing the information. When the reviewer finds any issues in the information you provided, you need to restart this process.

EAS Metadata uses a [**store.config.json**](./config.md#static-store-config) file to provide information instead of going through multiple forms in the app store dashboards. When it's time to update the app stores, you can push the store config to the app stores.

<Terminal cmd={['$ eas metadata:push']} />

> EAS Metadata can also instantly identify known app store restrictions that could trigger a rejection after a length review queue.

Reviewer: ```suggestion
EAS Metadata can also instantly identify known app store restrictions that could trigger a rejection after a lengthy review queue.
```

> When adding the store config file to your repository, you can collaborate with others to prepare the app submission.

Reviewer: ```suggestion
Adding the store config file to your repository enables you to collaborate with other team members to prepare the app submission.
```

Does the changed context fit here? It wasn't clear to me before how just by adding the store config I can collaborate with others to prepare the app submission?

Author: In this context, I assumed that you know what a git repo is. The collaboration comes from the part that you can add a file to git + work on it together, as opposed to 1 person going to the app dashboard and filling out all forms. If that makes sense 😄

Reviewer: Haha I thought one needed to be part of the organization to collaborate or something like that 😄 (My mind automatically went there).

I still think the changes suggested highlight the advantage a developer gets from it.

---
title: EAS Metadata
sidebar_title: Introduction
hideTOC: true
---

import { BoxLink } from '~/ui/components/BoxLink';
import { Callout } from '~/ui/components/Callout';
import { Terminal } from '~/ui/components/Snippet';

<Callout type="warning">
  EAS Metadata is in beta and subject to breaking changes.
</Callout>
<br />

> **EAS Metadata** is a tool that helps submit app store information to publish your app.

Reviewer 2: ```suggestion
**EAS Metadata** enables you to automate and maintain your app store presence from the command line.
```

You need to provide a lot of information to multiple app stores before your users can use your app. This information is often about complex topics that don't apply to your app. You have to start a lengthy review process after providing the information. When the reviewer finds any issues in the information you provided, you need to restart this process.

## Store config

> EAS Metadata aims to make this easier by using a [**store.config.json**](./config.md#static-store-config) file. You can provide all information in this store instead of going through multiple forms in the app store dashboards. When it's time to update the app stores, you can push the store config to the app stores.

Reviewer 2: This header is strange because it punctuates the previous paragraph, then the next paragraph continues by saying "make this easier".

Reviewer 2: Don't say "aims to make this easier", the feature does or doesn't do things.

Reviewer 2: I'd add a part about how you can store your metadata locally in git.

<Terminal cmd={['$ eas metadata:push']} />

> ## Built-in validation

Reviewer 2: I'd drop this header too

> EAS Metadata does not only get the information into the stores; it also tries to find common pitfalls that could cause app rejections. It does all this before sending anything to the app stores, saving you valuable time to spend on your project.

Reviewer 2: ```suggestion
EAS Metadata can also instantly identify known app store restrictions that could trigger a rejection after a length review queue.
```

Reviewer 2: specifically, dropping the "tries to find".
